00_Base/src/repository/OcpiLocationRepository.ts

```diff
@@ -77,6 +77,54 @@ export class OcpiLocationRepository extends SequelizeRepository<OcpiLocation> {
     );
   }
 
+  async createOrUpdateOcpiLocation(
+    location: Partial<OcpiLocation>,
+  ): Promise<OcpiLocation> {
+    if (location[OcpiLocationProps.citrineLocationId]) {
```

Comment:
i was trying to do good by the ocpi location props we had, but agreed i think it might just be difficult to read as is 😬 i will update and hold off on the props for now...